librad/src/net/protocol.rs

```diff
@@ -61,7 +61,7 @@ use crate::{
 #[derive(Clone, Debug)]
 pub enum ProtocolEvent<A> {
     Connected(PeerId),
-    Disconnecting(PeerId),
+    Disconnected(PeerId),
```

Comment:
Nit: I'd leave it as `Disconnecting` (progressive form), because that is what is happening -- the disconnect is only complete once all references to the connection are dropped. This might not be the case when the event is emitted -- f.ex. there could still be a `git` stream in-flight.